cmd/object-handlers.go

```diff
@@ -548,12 +549,20 @@ func (api objectAPIHandlers) getObjectHandler(ctx context.Context, objectAPI Obj
 			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL)
 			return
 		}
+		// `mc head` will close the connection, will make this error
```

Comment:
We need to move this type of check into IsNetworkOrHostDown(err) 

So that we can ignore printing the logs instead of introducing a goto jump